playbooks/ansible-test-cloud-integration-base/pre.yaml

```diff
@@ -0,0 +1,53 @@
+---
+- hosts: controller
+  tasks:
+    - name: Setup tox role
+      include_role:
+        name: tox
+      vars:
+        tox_extra_args: "-vv -- ansible-playbook -v playbooks/ansible-test-cloud-integration-base/files/bootstrap-vcsa.yaml"
+        tox_install_siblings: false
+        zuul_work_dir: "{{ zuul.projects['github.com/ansible/ansible-zuul-jobs'].src_dir }}"
+
+    - name: Setup base virtualenv_options
+      set_fact:
+        _virtualenv_options: "--python python{{ ansible_test_python }}"
+
+    - name: Enable --system-site-packages for virtualenv_options
+      set_fact:
+        _virtualenv_options: "{{ _virtualenv_options }} --system-site-packages"
+      when: ansible_os_family == "RedHat"
```

Comment:
I believe this is the issue for six, as we are using the system version inside the venv. My fix for this, is to drop --system-site-packages from virtualenv, and use https://pypi.org/project/selinux/

I tested it before and worked, I can open another PR to confirm network jobs don't break using it.

Reply:
Indeed. I will let the six thing aside for now. It will be addressed in another PR.

Reply:
Ok, it works fine without the `--system-site-packages`, thanks for the idea.

Reply:
Hum, Why should I? I don't need to pin on a specific six version anymore.

Reply:
This was originally dealing with selinux errors controller side, when updating things like file permissions. Given that it works now, makes me think, this isn't needed any more on centos-8 but is something we could hit in fedora.

Lets leave this for now, the longer term fix is to drop this, but I think we also need selinux package for selinux python bindings inside the venv.  I'll work on testing that this morning.